CONTRIBUTING.md

```diff
@@ -177,6 +177,32 @@ which shows the baseline image, the generated image, the diff and the json mocks
 
 To view the results of a run on CircleCI, download the `build/test_images/` and `build/test_images_diff/` artifacts into your local repo and then run `npm run start-image_viewer`.
 
+### Writing selection tests
+Keep in mind that the selection coordinates are relative to the top-left corner of the plot, including the margins. To produce a reliable selection test, 
+it may be necessary to to fix the width, height, margins, X axis range and Y axis range of the plot. For example:
+
+```
+Plotly.newPlot(gd,
+    [
```

Comment:
Although not a linting mistake, we prefer formatting blocks like this one like this:

```js
Plotly.newPlot(gd, [{
    // ...
}], {
    // ...
});